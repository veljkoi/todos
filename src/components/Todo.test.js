import React from 'react';
import renderer from 'react-test-renderer';

import Todo from './Todo';

describe('<Todo/>', () => {

	test('renders with line-trough if completed', () => {
		const todo = {
			onClick: () => {},
			text: 'test',
			completed: true
		};
		const tree = renderer.create(<Todo {...todo}/>).toJSON();
		expect(tree).toMatchSnapshot();
	});

});