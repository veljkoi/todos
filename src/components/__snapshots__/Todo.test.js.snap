// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Todo/> renders with line-trough if completed 1`] = `
<li
  onClick={[Function]}
  style={
    Object {
      "textDecoration": "line-through",
    }
  }
>
  test
</li>
`;
